pages/content/shared/footer.html

```diff
@@ -0,0 +1,20 @@
+$view: /layouts/blank.j2
+---
+<style amp-custom>
+  {% do doc.styles.addCssFile('css/base.css', -5) %}
+  {% do doc.styles.addCssFile('css/components/atoms/text.css', -3) %}
+  {% do doc.styles.addCssFile('css/components/atoms/anchor.css', -2) %}
+  {% do doc.styles.addCssFile('css/components/atoms/icon.css', -1) %}
+
+  {{ doc.styles.emit() }}
+</style>
+
+{% block icons %}
+<svg style="position: absolute; width: 0; height: 0; overflow: hidden;" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+  <defs>
+    {{ doc.icons.emit() }}
+  </defs>
+</svg>
+{% endblock %}
```

Comment:
Can this SVG markup be removed from here? It's the same I believe, so we only need that once (in `header.html`).